### PR TITLE
NCD - Medical History

### PR DESCRIPTION
**Pull request by anvmn**
#400
#401
#402
#403

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-08-30 19-18

**Comment by anvmn** [comment]
Peek 2022-08-31 12-49

**Comment by anvmn** [comment]
Peek 2022-08-31 14-00

**Comment by anvmn** [comment]
Peek 2022-09-01 12-06

**Comment by anvmn** [comment]
Peek 2022-09-01 13-15

**Comment by anvmn** [comment]
@AronNovak 

Please review. There are several @todo to be completed in follow up PRs.

File: client/src/elm/Pages/NCD/Activity/Update.elm [hunk 1]
```diff
@@ -487,3 +517,365 @@ update currentDate id db msg model =
...
+            in
+            ( { model | medicalHistoryData = updatedData }
+            , Cmd.none
+            , []
+            )
+
+        SetHypertensionPredecessor predecessor ->
+            let
+                _ =
+                    Debug.log "SetHypertensionPredecessor" predecessor
```
Comment (AronNovak) [inline client/src/elm/Pages/NCD/Activity/Update.elm]: shall we remove  ̇`Debug.log` from merged PRs?

File: client/src/elm/Pages/NCD/Activity/Update.elm [hunk 2]
```diff
@@ -487,3 +517,365 @@ update currentDate id db msg model =
...
+            in
+            ( { model | medicalHistoryData = updatedData }
+            , Cmd.none
+            , []
+            )
+
+        SetHeartProblemPredecessor predecessor ->
+            let
+                _ =
+                    Debug.log "SetHeartProblemPredecessor" predecessor
```
Comment (AronNovak) [inline client/src/elm/Pages/NCD/Activity/Update.elm]: same as above

File: client/src/elm/Translate.elm [hunk 3]
```diff
@@ -8446,6 +8729,33 @@ translationSet trans =
                     , kinyarwanda = Just "Nta kimenyetso na kimwe"
                     }
 
+        Predecessor predecessor ->
+            case predecessor of
+                PredecessorFather ->
+                    { english = "Father"
+                    , kinyarwanda = Nothing
```
Comment (AronNovak) [inline client/src/elm/Translate.elm:8736]: How do we handle translation flow? Can we export not yet translated strings to translators from code at a later time?

**Review by AronNovak — APPROVED** [review]
Looks good mostly, a few questions above

File: client/src/elm/Translate.elm (hunk 3 above)
Comment (anvmn) [reply client/src/elm/Translate.elm:8736]: @AronNovak 
We do not hold deployments till everything is translated.
Translations are completed with follow up PRs.

**Pull request merged**